### PR TITLE
Added basic carousel

### PR DESCRIPTION
**Pull request by sudhakg**

### CODE REVIEW

File: client/src/components/relatedItems/RelatedItems.jsx [hunk 1]
```diff
@@ -1,19 +1,120 @@
...
   render() {
     return (
       <div>
-        <h2>Related Products</h2>
-        <div>CardsList</div>
-        <div>YourOutfitList</div>
+        <h1>Related Products</h1>
+        <div>
+        <button onClick={this.handle_previous}>Previous</button>
+        <button onClick={this.handle_next}>Next</button>
```
Comment (sudhakg) [inline client/src/components/relatedItems/RelatedItems.jsx:83]: I will replace the buttons eventually. I just have it for now to move the cards in carousel left and right

File: webpack.config.js [hunk 2]
```diff
@@ -16,6 +16,13 @@ module.exports = {
           loader: 'babel-loader',
         },
       },
+      {
+        test: /\.css$/,
+        use: [
+          'style-loader',
+          'css-loader'
+        ]
+      }
```
Comment (sudhakg) [inline webpack.config.js:25]: I  had to configure without which the component was not picking up css even though the .css file in the same package as component

File: client/src/components/relatedItems/RelatedItems.jsx [hunk 3]
```diff
@@ -1,19 +1,120 @@
...
+            <Item itemNumber={5}/>
+          </div>
+        </div>
       </div>
-    );
+    )
+  }
+}
+
+const styles = {
```
Comment (farhanali3193) [inline client/src/components/relatedItems/RelatedItems.jsx]: @sudhakg Can the styles object be moved to the css file? Or does it break something?

File: webpack.config.js (hunk 2 above)
Comment (farhanali3193) [reply webpack.config.js:25]: Understood, now we can use individual stylesheets for our widgets.

File: client/src/components/relatedItems/RelatedItems.jsx (hunk 1 above)
Comment (farhanali3193) [reply client/src/components/relatedItems/RelatedItems.jsx:83]: Got it. Can you also fix the failing tests?

**Pull request merged**